test/config/install_coreos_maximum_QuantaT41.json

```diff
@@ -0,0 +1,51 @@
+{
+    "bootstrap-payload": {
+        "name": "Graph.InstallCoreOS",
+        "options": {
+            "defaults": {
+                "hostname": "rackhdnode",
```

Comment:
@hohene Did you get an answer on the above from George?  I noticed you approved the changes.